text/0000-addon-version-compatibility.md

```diff
@@ -0,0 +1,79 @@
+- Start Date: 2015-09-04
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This feature adds a standard of marking addons with version compatibility to core dependencies such as Ember, Ember-CLI, Ember-Data, etc.
+
+# Motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+The Ember Addon ecosystem is booming, but it is also becoming increasingly crowded with a deluge of information. 
+Sites like Ember Addons and Ember Observer attempt to lessen the information overload by categorizing and scoring addons.
+However, it is still difficult to identify addons that are outdated or addons that only work past a certain version of Ember, for example.
+This RFC details a proposal to standardize tagging of addon compatibility by their authors, so that services such as Ember Addons or Ember Observer
+can consume that information to surface to developers in an easy-to-understand format. This could also enable features such as Ember-CLI warning users
+when they attempt to `ember install` a package that is listed as incompatible with the core package versions of their application.
+
+# Detailed design
+
+This is accomplished by adding a `compatibility` tag to the `ember-addon` section of an addon's `package.json`. 
+This tag should contain a hash of package names that are associated with range strings identifying the compatible versions.
+Version ranges should be formatted as outlined by [Node-SemVer](https://github.com/npm/node-semver).
+ 
+Example `package.json`:
+
+```json
+{
+    "name": "ember-my-addon",
+    "version": "0.0.1",
+    "description": "My Ember Addon",
+    "keywords": [
+        "ember-addon"
+    ],
+    "ember-addon": {
+        "configPath": "tests/dummy/config",
+        "demoURL": "username.github.io/ember-my-addon",
+        "compatibility": {
+            "ember": "^1.12.0",
+            "ember-cli": "1.13.1 - 1.13.8",
+            "ember-data": "1.0.0-beta.18"
+        }
+    }
+}
+```
+
+The Ember-CLI blueprint for new addons could possibly include some default compatibility that 
+appropriately indicates the compatibility of a the current Ember releases with other versions.
+Authors of addons should then update their package.json to reflect the compatibility of their package.
+
+The usage of this feature is not restricted to core Ember libraries, obviously. 
+Addon authors could use it to mark other dependencies which they have strict compability with.
+
+# Drawbacks
+
+This is still a manual process depending on other maintainers, and if Ember-CLI adds this to the default blueprint, 
+we may see a slew of addons that use the default but never get updated as things become incompatible.
+
+# Alternatives
+
```

Comment:
I believe the plan (and my current thought) is for us to use `peerDependencies` and make `ember install` more intelligent to resolve & download, or detect conflict and report.

Adding an additional concept like `compatibility` should be reserve if ^^ doesn't work for some reason. So far no blockers have been noticed.